### PR TITLE
pip: add German translation

### PR DESCRIPTION
**Pull request by dadav**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.de/common/pip.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# pip
+
+> Python package manager.
+> Einige Unterbefehle wie `pip install` sind separat dokumentiert.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket (siehe `pip install` für weitere Beispiele):
+
+`pip install {{paketname}}`
```
Comment (navarroaxel) [inline pages.de/common/pip.md:9]:
```suggestion
`pip install {{paket_name}}`
```

File: pages.de/common/pip.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Upgrade ein Paket:
+
+`pip install -U {{paketname}}`
```
Comment (navarroaxel) [inline pages.de/common/pip.md:13]:
```suggestion
`pip install -U {{paket_name}}`
```

File: pages.de/common/pip.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Deinstalliere ein Paket:
+
+`pip uninstall {{paketname}}`
```
Comment (navarroaxel) [inline pages.de/common/pip.md:17]:
```suggestion
`pip uninstall {{paket_name}}`
```

File: pages.de/common/pip.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Speichere eine Liste der installierten Pakete in eine Datei:
+
+`pip freeze > {{requirements.txt}}`
+
+- Zeige die installierten Pakete an:
+
+`pip show {{paketname}}`
```
Comment (navarroaxel) [inline pages.de/common/pip.md:25]:
```suggestion
`pip show {{paket_name}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
@dadav remember that you can send multi translations in one PR.

File: pages.de/common/pip.md (hunk 1 above)
Comment (dadav) [reply pages.de/common/pip.md:9]: Hi Navarroaxel, in german it's `Paketname`, do you still want me to split it?

**Comment by dadav** [comment]
> @dadav remember that you can send multi translations in one PR.

Sorry, I didn't know this is welcome. I'm used to make one PR for one `feature` :)

File: pages.de/common/pip.md (hunk 1 above)
Comment (dadav) [reply pages.de/common/pip.md:9]: Let's keep it the same as in `pip-install` (@marchersimon approved it there)

File: pages.de/common/pip.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+> Python package manager.
+> Einige Unterbefehle wie `pip install` sind separat dokumentiert.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket (siehe `pip install` für weitere Beispiele):
+
+`pip install {{paketname}}`
+
+- Upgrade ein Paket:
```
Comment (marchersimon) [inline pages.de/common/pip.md:11]: What do you think about `aktualisiere` here?

File: pages.de/common/pip.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+`pip install -U {{paketname}}`
+
+- Deinstalliere ein Paket:
+
+`pip uninstall {{paketname}}`
+
+- Speichere eine Liste der installierten Pakete in eine Datei:
```
Comment (marchersimon) [inline pages.de/common/pip.md:19]:
```suggestion
- Speichere eine Liste aller installierten Pakete in eine Datei:
```
What do you think?

File: pages.de/common/pip.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+`pip freeze > {{requirements.txt}}`
+
+- Zeige die installierten Pakete an:
```
Comment (marchersimon) [inline pages.de/common/pip.md:23]: Shouldn't that be `Zeige Informattionen über ein installiertes Paket an`?

Comment (dadav) [reply pages.de/common/pip.md:23]: You are right, I'll change it. Thank you

File: pages.de/common/pip.md (hunk 5 above)
Comment (dadav) [reply pages.de/common/pip.md:11]: Good idea, I'll change it! Thank you

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Pull request merged**